javascript/javascript_in_the_real_world/linting.md

```diff
@@ -25,10 +25,18 @@ The style guides we mentioned above are full of really helpful advice for format
 1. This article on [JavaScript linters](https://gomakethings.com/javascript-linters/) gets right to the point... start here!
 1. Read this article that [goes a little further](https://hackernoon.com/how-linting-and-eslint-improve-code-quality-fa83d2469efe) by discussing exactly *how* linters do what they do.
 
-There are multiple options for linting your JavaScript, but the most popular (and most common in the industry) is [eslint](https://eslint.org/). Getting it installed and the initial set-up is straightforward.
+There are multiple options for linting your JavaScript, but the most popular (and most common in the industry) is [ESLint](https://eslint.org/). Getting it installed and the initial set-up is straightforward.
 
 1. [The official 'Getting Started' page](https://eslint.org/docs/user-guide/getting-started) is a good place to start. It covers installation and basic setup. The basic way to use this tool is to run the `eslint` command in your terminal with a specific file.
-1. There is an [ESLint extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) which can lint your files as you write, without you needing to rerun the `eslint` command every time. If your project also contains an ESLint configuration file, the extension will automatically use those rules for that project. Here is a [tutorial on using the ESLint extension in Visual Studio Code](https://www.digitalocean.com/community/tutorials/linting-and-formatting-with-eslint-in-vs-code).
+1. There is an [ESLint extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) with which you can get automatic lint highlighting for your files as you write, without you needing to rerun the `eslint` command every time. If your project also contains an ESLint configuration file, the extension will automatically use those rules for that project.
```

Comment:
### DO NOT COMMIT THIS SUGGESTION

```suggestion
   - DO NOT COMMIT THIS. THIS IS JUST TO SHOW THE POSITION OF THE SUGGESTION
1. There is an [ESLint extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) with which you can get automatic lint highlighting for your files as you write, without you needing to rerun the `eslint` command every time. If your project also contains an ESLint configuration file, the extension will automatically use those rules for that project.
```

I think it would be beneficial to add a small sub-bullet linking to the ["Configure ESLint" page](https://eslint.org/docs/latest/use/configure/) and potentially even adding a note about [ignoring files](https://eslint.org/docs/latest/use/configure/ignore) (regarding the difference between `ignores` being a lone object property vs in a config object with multiple).

Tbh I'm not 100% sure about including the `ignores` thing, or at least the best way to not be more confusing than not. We should definitely link to the config overview page so people are aware of where they can go to learn about amending the ESLint config.

Reply:
Hmm, I see the merit in adding a sub-bullet for the configuration page.

As for the part about ignoring files, I'm not sure either.
You know better about the later parts of the curriculum, so if you feel that knowing about this peculiar behaviour of `ignores` is something that may help learners in the future projects/assignments, then we can go ahead and add it.

If it's something that doesn't see much use within the curriculum, but is common in non-curriculum environment, then perhaps we could trust the learners to look for it, since the top result for `eslint global ignore` is the `Ignore Files` page in ESLint docs.


> In your eslint.config.js file, if an ignores key is used without any other keys in the configuration object, then the patterns act as global ignores

The relevant reading on `ignores` special behaviour is just a single line, so I don't think we'd go wrong either way. I'm fine with whatever your call is.

Reply:
Let's omit it for now and see how it goes. I'm going to be PRing an improvement to ESLint docs if there isn't already one open, so hopefully it'll be improved soon enough, be that from my changes or someone elses.

Reply:
Do the changes look good?

Reply:
> Do the changes look good?

I'll have a proper review in a bit. For future reference, if changes are requested and you're now in a state where it's ready for re-review, you can re-request a review from that maintainer by hitting the rerequest button in the top right:
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/43245282-521c-4b39-a70e-6af2956b59d6)

Reply:
I clicked the button :D